src/Components/Articles.jsx

```diff
@@ -17,18 +18,21 @@ const Articles = () => {
   ) : (
     <div className="articles">
       <h1>Articles</h1>
-      <ul className="articles-list">
         {articles.map((article) => {
           return (
-            <li className="article-item" key={article.article_id}>
-              <p className="article-title">{article.title}</p>
-              <p className="article-topic">{article.topic}</p>
-              <p className="article-author">{article.author}</p>
+            <Link to={`/articles/${article.article_id}`}>
+            <div className="articles-list">
+            <li className="articles-item" key={article.article_id}>
+                <img className="articles-img"src={article.article_img_url} alt={article.title}/>
+              <p className="articles-title">{article.title}</p>
+              <p className="articles-topic">{article.topic}</p>
+              <p className="articles-author">{article.author}</p>
             </li>
+            </div>
+            </Link>
```

Comment:
I really like that the whole card is a link, this plays really well especially in mobile platforms 

Consider refactoring this code to it's own component